docs/apache-airflow-providers-cncf-kubernetes/operators.rst

```diff
@@ -77,8 +77,8 @@ You can print out the Kubernetes manifest for the pod that would be created at r
 Argument precedence
 ^^^^^^^^^^^^^^^^^^^
 
-When building the pod object, there may be overlap between KPO params, pod spec, template and airflow connection.
-In general, the order of precedence is KPO argument > full pod spec > pod template file > airflow connection.
+When KPO defines the pod object, there may be overlap between the :class:`~airflow.providers.cncf.kubernetes.operators.pod.KubernetesPodOperator` arguments.
+In general, the order of precedence is ``full_pod_spec``, ``pod_template_file``, or otherwise the ``V1Pod`` default.
```

Comment:
```suggestion
In general, the order of precedence is ``full_pod_spec``, ``pod_template_file``, or otherwise the ``V1Pod`` default.
```
```suggestion
In general, the order of precedence is ``full_pod_spec``, ``pod_template_file``, followed by ``V1Pod``, by default.
```